File: aspnet-in-practice/14/VB/Cache/ObjectModel/Customer.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web

<Serializable>
Public Class Customer
	Private _savedDate As DateTime = DateTime.Now

	Public Sub New()
	End Sub

	Public Property FirstName() As String
	Public Property LastName() As String
	Public ReadOnly Property SavedDate() As DateTime
		Get
			Return (_savedDate)
		End Get
	End Property

End Class

File: aspnet-in-practice/14/CS/Cache/Cache/ObjectModel/Customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

[Serializable]
public class Customer
{
	DateTime _savedDate = DateTime.Now;

	public Customer()
	{
	}

	public string FirstName { get; set; }
	public string LastName { get; set; }
	public DateTime SavedDate
	{
		get
		{
			return _savedDate;
		}

	}
}